database/other/podcasts.json

```
@@ -19,6 +19,12 @@
     "url": "https://stackoverflow.blog/podcast/",
     "category": "other",
     "subcategory": "podcasts"
+  },
+  {
+    "name": "Overpowered",
+    "description": " In Overpowered, the hosts Varun Mayya & Tanmay Bhat talk about everything that you can imagine about AI, helping you get insights into how this piece of tech can be integrated in various domains.",
```

Good resource. Just this starting space could to be removed.

```suggestion
    "description": "In Overpowered, the hosts Varun Mayya & Tanmay Bhat talk about everything that you can imagine about AI, helping you get insights into how this piece of tech can be integrated in various domains.",
```